### PR TITLE
Rename Eta

### PR DESCRIPTION
**Pull request by rebeccafair**
Updating as a result of https://github.com/pace-neutrons/Euphonic/issues/114. There are also some small changes to allow Euphonic's warnings to be propagated.

CI tests will fail until Euphonic's changes are in master as the CI tests only test against Euphonic's master branch.

### CODE REVIEW

File: euphonic_sqw_models/euphonic_wrapper.py [hunk 1]
```diff
@@ -51,14 +60,24 @@ class CoherentCrystal(object):
...
+    defaults = {'debye_waller': None, 'debye_waller_grid': None,
+                'temperature': 0.0 * ureg('K'), 'bose': True,
+                'negative_e': False, 'conversion_mat': None,
+                'chunk': 5000, 'lim': np.inf,
+                'scattering_lengths': 'Sears1992', 'weights': None,
+                'asr': None, 'dipole': True,
+                'dipole_parameter': 1.0 ,'eta_scale': 1.0,
+                'splitting': True, 'insert_gamma': False,
+                'reduce_qpts': True, 'use_c': None,
+                'n_threads': None, 'verbose': True}
```
Comment (cmarooney-stfc) [inline euphonic_sqw_models/euphonic_wrapper.py]: The formatting here looks as if it only went half-way. Alignment of the second default on each line to make a nice column would also make it easier to read and determine, say, that ureg('K') went with temperature and not with bose.

File: euphonic_sqw_models/euphonic_wrapper.py [hunk 2]
```diff
@@ -142,9 +161,9 @@ def _calculate_phonon_modes(self, qpts):
         if self.force_constants is None:
             raise RuntimeError('Force constants model not set')
         return self.force_constants.calculate_qpoint_phonon_modes(qpts,
-            weights=self.weights, asr=self.asr, dipole=self.dipole, eta_scale=self.eta_scale,
-            splitting=self.splitting, insert_gamma=self.insert_gamma, reduce_qpts=self.reduce_qpts,
-            use_c=self.use_c, n_threads=self.n_threads)
+            weights=self.weights, asr=self.asr, dipole=self.dipole, dipole_parameter=self.dipole_parameter,
+            eta_scale=self.eta_scale, splitting=self.splitting, insert_gamma=self.insert_gamma,
+            reduce_qpts=self.reduce_qpts, use_c=self.use_c, n_threads=self.n_threads)
```
Comment (cmarooney-stfc) [inline euphonic_sqw_models/euphonic_wrapper.py:176]: Noting that eta_scale continues to exist despite eta being deprecated. Is this just the deprecated option? Apparently yes looking at PR-170. Would you prefer to rename it to dipole_paramenter and then just pass dipole_parameter.

File: euphonic_sqw_models/euphonic_wrapper.py (hunk 1 above)
Comment (rebeccafair) [reply euphonic_sqw_models/euphonic_wrapper.py]: I don't think I've ever seen that kind of alignment in Python, I can't see anything specifically forbidding it in PEP8 but I can see that it discourages similar things (e.g. last part of [this section](https://www.python.org/dev/peps/pep-0008/#pet-peeves)). I know the rest of the file doesn't follow PEP8 but I'm hoping to fix that at some point...

What I can do though is put one dictionary entry on each line, that will probably make it a lot easier to read.

File: euphonic_sqw_models/euphonic_wrapper.py (hunk 2 above)
Comment (rebeccafair) [reply euphonic_sqw_models/euphonic_wrapper.py:176]: Yes `eta_scale` here is the deprecated option. I wanted it to still be used so if someone uses `eta_scale` from `euphonic_sqw_models` (or `Horace-Euphonic-Interface`) they will get the deprecation message. If we just silently rename `eta_scale` -> `dipole_parameter` internally users won't know they're supposed to be using something else. I didn't want to have to manually raise another `DeprecationWarning` from here either, would rather let Euphonic do it.

**Pull request merged**